luigi/task.py

```diff
@@ -146,6 +146,8 @@ class MyTask(luigi.Task):
     #: Only works when using multiple workers.
     worker_timeout = None
 
+    config = {}
```

Comment:
Please comment this. Also the name `config` is both short and unclear. I imagine something very explicit like `per_task_retry_policy`, or perhaps the shorter `retry_policy` or `retry_policy_dict`. A non-luigi user reading luigi code would then have a chance to understand this.

There probably would be some user who already use `config`